yal/tests/test_utils.py

```diff
@@ -122,3 +123,21 @@ def test_is_integer():
 
     obj = 1234
     assert utils.is_integer(obj)
+
+
+class PhoneNumberTest(unittest.TestCase):
```

Comment:
Are we inheriting from `unittest.TestCase` to use `assertRaises`?

Reply:
No i like to group all related test under a class